robocorp-code/vscode-client/src/locators.ts

```diff
@@ -8,25 +13,41 @@ import { OUTPUT_CHANNEL } from './channel';
 export async function copySelectedToClipboard(locator?: LocatorEntry) {
     let locatorSelected: LocatorEntry | undefined = locator || getSelectedLocator();
     if (locatorSelected) {
-        vscode.env.clipboard.writeText(locatorSelected.name);
+        env.clipboard.writeText(locatorSelected.name);
     }
 }
 
 export async function removeLocator(locator?: LocatorEntry) {
+    // Confirmation dialog button texts
+    const DELETE = 'Delete';
```

Comment:
Instead of `getSelectedLocator()`, shouldn't it use the one received as a parameter (if available)?

Reply:
let locatorSelected: LocatorEntry | undefined = locator || getSelectedLocator();
The || notation should take care of prioritizing the parameter value.. could be ?? as well to strictly compare it with undefined (optional value)

Reply:
You're right, sorry, my bad upon reviewing.